perfmetrics/scripts/ml_tests/pytorch_dino_model/setup_container.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+wget -O go_tar.tar.gz https://go.dev/dl/go1.19.4.linux-amd64.tar.gz
+rm -rf /usr/local/go && tar -C /usr/local -xzf go_tar.tar.gz
+export PATH=$PATH:/usr/local/go/bin
+
+git clone https://github.com/raj-prince/gcsfuse.git
+cd gcsfuse
+git checkout build_script_pytorch
+go build .
+cd -
+
+echo "Mounting GCSFuse..."
+nohup /pytorch_dino/gcsfuse/gcsfuse --type-cache-ttl=1728000s \
+        --stat-cache-ttl=1728000s \
+        --stat-cache-capacity=1320000 \
+        --stackdriver-export-interval=60s \
+        --implicit-dirs \
+        --disable-http2 \
+        --max-conns-per-host=100 \
+        --debug_fs \
+        --debug_gcs \
+        --log-file logs.txt \
```

Comment:
nit: create a directory with name 'logs' inside gcsfuse directory or directory with name 'gcsfuse' alongside 'gcsfuse_data' and store gcsfuse logs inside it for better organisation.